From: =?utf-8?q?Rog=C3=A9rio_Brito?= <rbrito@ime.usp.br>
Date: Sat, 25 Jun 2016 13:53:01 -0300
Subject: Update list of supported sites
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

This patch is the result of the automatic generation of supported sites by
youtube-dl.  The order here *may* possibly depend on the locale of the
system where the list is generated, but I have not investigated it.

Author: Rogério Brito <rbrito@ime.usp.br>

---

Forwarded: no
Last-Update: 2016-06-25
---
 docs/supportedsites.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/supportedsites.md b/docs/supportedsites.md
index 891499f..470fd40 100644
--- a/docs/supportedsites.md
+++ b/docs/supportedsites.md
@@ -44,8 +44,8 @@
  - **appletrailers:section**
  - **archive.org**: archive.org videos
  - **ARD**
- - **ARD:mediathek**
  - **ARD:mediathek**: Saarländischer Rundfunk
+ - **ARD:mediathek**
  - **arte.tv**
  - **arte.tv:+7**
  - **arte.tv:cinema**
